input/fsh/au-erequesting-pathrequest.fsh

```diff
@@ -0,0 +1,23 @@
+Profile: AUeRequestingPathologyRequest
+Parent: AUeRequestingServiceRequest
+Id: au-erequesting-pathrequest
+Title: "AU eRequesting Pathology Request"
+Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about pathology test requests for a patient. It is based on the [AU eRequesting ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
+
+* ^status = #draft
+
+* ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
+
+* . ^short = "A pathology test request"
+* . ^definition = "A request for a pathology service."
+
+* category = http://snomed.info/sct#108252007
```

Comment:
I was incorrect. Pattern is not how we approach these. The coding should be sliced. See https://hl7.org.au/fhir/4.1.2-preview/StructureDefinition-au-pathologyresult.html

Reply:
Yes. Added a slice.